include/common_structs.h

```diff
@@ -337,17 +337,21 @@ typedef struct {
     // 1st place is 3 points, 2nd is 2, etc.
     // Lowest dibit is the Mushroom Cup, 2nd dibit is the Flower Cup, etc
     u8 grandPrixPoints[4];
+    u8 soundMode;
     // Its unknown what these bytes are used for
-    // They appear related to the Grand Prix points in some capacity but I can't figure out how
-    u8 unknownBytes1[4];
+    u8 unknownBytes[3];
     // For some reason there's 2 entries covering 4 cups
```

Comment:
Could these unk ones just be pointers back to the save data struct. Although, that would be pretty obvious because in-game debugger would show as variable address in that spot.

Reply:
I don't think so, they're purpose seems related to being a hash of sorts for the save data.
```func_800B492C``` seems to calculate something akin to a hash of grandPrixPoints and the screenMode. That value is stored in ```unknownBytes[1]``` by the looks of it.
```unknownBytes[2]``` seems to be somewhat similar, no idea what ```unknownBytes[0]``` could be for though.